### PR TITLE
Added documentation on brew

### PR DESCRIPTION
**Pull request by szechy**

### CODE REVIEW

File: _commands/os_x/brew.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Updated Homebrew from 86fc508 to 5531fd9.
+Updated 1 tap (homebrew/dupes).
+==> New Formulae
+...
+==> Updated Formulae
+...
+==> Deleted Formulae
+...
+~~~
+##### Break it down
```
Comment (themoh) [inline _commands/os_x/brew.md]: screen shot 2016-03-17 at 7 31 44 pm

Fix the leading #'s

**Comment by themoh** [comment]
Fix missing bullet points under the "Break it down"s

File: _commands/os_x/brew.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+---
+---
+
+brew
+--
+
+`brew`, or known by its longer name as Homebrew, bills itself as the "missing package manager for OS X". Here's an example of installing a package:
```
Comment (themoh) [inline _commands/os_x/brew.md]: Remove the last sentence ("Here's an example...")

File: _commands/os_x/brew.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+--
+
+`brew`, or known by its longer name as Homebrew, bills itself as the "missing package manager for OS X". Here's an example of installing a package: 
+
+~~~
+$ brew install wget
+==> Downloading https://homebrew.bintray.com/bottles/wget-1.17.1.el_capitan.bottle.1.tar.gz
+Already downloaded: /Library/Caches/Homebrew/wget-1.17.1.el_capitan.bottle.1.tar.gz
+==> Pouring wget-1.17.1.el_capitan.bottle.1.tar.gz
+🍺  /usr/local/Cellar/wget/1.17.1: 9 files, 1.5M
```
Comment (themoh) [inline _commands/os_x/brew.md]: include an example of a package that you don't already have downloaded (uninstall and reinstall wget)

**Comment by szechy** [comment]
Updated with an extra commit to address @themoh's suggestions.

File: _commands/os_x/brew.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+##### Break it down
+`brew upgrade` upgrades all of ones' packages to the latest releases.
+
+#### `brew list`
+~~~
+$ brew list
+autoconf	cowsay		gdbm		libtool		openssl		rbenv		sqlite
+automake	dfu-util	gettext		libusb		pkg-config	readline	tmux
+boost		fortune		gnu-getopt	mongodb		postgresql	ruby-build	valgrind
+coreutils	gdb			libevent	node		python		scons		xz
```
Comment (themoh) [inline _commands/os_x/brew.md]: fix indenting

File: _commands/os_x/brew.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+Updated 1 tap (homebrew/dupes).
+==> New Formulae
+...
+==> Updated Formulae
+...
+==> Deleted Formulae
+...
+~~~
+
+##### Break it down
```
Comment (themoh) [inline _commands/os_x/brew.md]: This still shows the leading #'s

**Comment by themoh** [comment]
Bullet points still aren't showing

**Comment by szechy** [comment]
Added bullet points as @themoh suggested.

File: _commands/os_x/brew.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+Various Linux distributions include package managers for installing software out of the box (e.g. `apt-get` for Ubuntu), and `brew` is a third-party effort to replicate the same for OS X. 
+
+To install `brew`, the [official Homebrew website](http://brew.sh) directs you to run this command, which incidentally explains itself as it runs:
+
+~~~
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+~~~
+
+As an example of `brew` usage, here's installing [Node.js](https://nodejs.org/):
```
Comment (mterwill) [inline _commands/os_x/brew.md]: s/Node/rsync/

**Comment by mterwill** [comment]
Looks good! :+1:

**Comment by szechy** [comment]
Should be good now!

**Pull request merged**